packages/app/components/Questions/Queue/TAQueueDetailButtons.tsx

```diff
@@ -161,16 +228,34 @@ export default function TAQueueDetailButtons({
             <CantFindButton shape="circle" icon={<CloseOutlined />} />
           </Tooltip>
         </Popconfirm>
-        <Tooltip title="Finish Helping">
+        <Tooltip
+          title={
+            question.isTaskQuestion
+              ? tasksSelectedForMarking.length > 0
```

Comment:
Is there a way we can reduce complexity of this logic? More than condition seems excessive?